lib/uklibparam/include/uk/libparam.h

```diff
@@ -187,18 +130,16 @@ UK_CTOR_PRIO(UK_LIBPARAM_LIBDESC_CTOR, 0);
 
 #define __UK_LIBPARAM_PARAM_DEFINE(arg_var, arg_addr, arg_type, arg_count, \
 				   arg_desc)				\
-	static struct uk_libparam_param __UK_LIBPARAM_PARAM_NAME(arg_var) = { \
-		.name  = STRINGIFY(arg_var),				\
-		.desc  = arg_desc,					\
-		.type  = _UK_LIBPARAM_PARAM_TYPE_##arg_type,		\
-		.count = arg_count,					\
-		.addr  = arg_addr					\
+	static const struct uk_libparam_param				\
+	__used __section("." STRINGIFY(UK_LIBPARAM_PARAMSECTION_NAME))	\
+	__align(8) __UK_LIBPARAM_PARAM_NAME(arg_var) = {		\
```

Comment:
I wonder if it's worth surrounding every argument with `( )`? :thinking:

Reply:
I think it should be okay as is as we don't perform any operations.